src/web/routes.rs

```diff
@@ -31,6 +31,11 @@ pub(super) fn build_routes() -> Routes {
 
     routes.static_resource("/-/static/:single", super::statics::static_handler);
     routes.static_resource("/-/static/*", super::statics::static_handler);
+    routes.static_resource(
```

Comment:
we could argue this should be `internal_page` so we can differentiate local static file serving from this handler, which servers rustdoc specific files. 

What do you think @Nemo157 ?

Reply:
Agreed. I think we might be able to remove the `static_resource` method and just have everything be an `internal_page`, there's not _that_ many paths that it would be bad to split them out to separate metrics.